schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE "patients" (
+  "id" integer PRIMARY KEY,
+  "name" varchar,
+  "date_of_birth" date
+);
+
+CREATE TABLE "medical_histories" (
+  "id" integer PRIMARY KEY,
+  "admitted_at" timestamp,
+  "patient_id" integer,
+  "status" varchar
+);
+
+CREATE TABLE "invoices" (
+  "id" integer PRIMARY KEY,
+  "total_amount" decimal,
+  "generated_at" timestamp,
+  "payed_at" timestamp,
+  "medical_histories_id" integer
+);
+
+CREATE TABLE "invoices_items" (
+  "id" integer PRIMARY KEY,
+  "unit_price" decimal,
+  "quantity" integer,
+  "total_price" decimal,
+  "invoice_id" integer,
+  "treatment_id" integer
+);
+
+CREATE TABLE "treatments" (
+  "id" integer PRIMARY KEY,
+  "type" varchar,
+  "name" varchar
+);
+
+ALTER TABLE "medical_histories" ADD FOREIGN KEY ("patient_id") REFERENCES "patients" ("id");
+
+ALTER TABLE "invoices" ADD FOREIGN KEY ("medical_histories_id") REFERENCES "medical_histories" ("id");
+
+ALTER TABLE "invoices_items" ADD FOREIGN KEY ("invoice_id") REFERENCES "invoices" ("id");
+
+ALTER TABLE "invoices_items" ADD FOREIGN KEY ("treatment_id") REFERENCES "treatments" ("id");
+
+CREATE TABLE "treatments_medical_histories" (
+  "treatments_id" integer,
+  "medical_histories_id" integer,
+  PRIMARY KEY ("treatments_id", "medical_histories_id")
+);
+
+ALTER TABLE "treatments_medical_histories" ADD FOREIGN KEY ("treatments_id") REFERENCES "treatments" ("id");
+
+ALTER TABLE "treatments_medical_histories" ADD FOREIGN KEY ("medical_histories_id") REFERENCES "medical_histories" ("id");
```

Comment:
- [ ] 👍 You've made great progress in adding tables as suggested. However, it's important to note that, as per the project requirement, you should add indexes to all the FOREIGN KEYS in your database. By adding indexes to all the FOREIGN KEYS, you not only align with the project specifications but also enhance the overall performance of your database. Keep up the good work! 👍🗃️

![image](https://github.com/EvansSnave/database-from-diagram/assets/55840999/5c0097e1-0ed6-47f2-a6c6-b8abebfa92c3)